xml/System.Threading.Tasks/Task.xml

```diff
@@ -4470,13 +4470,13 @@ Task t Status: RanToCompletion, Result: 42
 > [!NOTE]
 >  The call to <xref:System.Threading.Tasks.Task.WhenAll%28System.Collections.Generic.IEnumerable%7BSystem.Threading.Tasks.Task%7D%29> method does not block the calling thread.  
   
- If any of the supplied tasks completes in a faulted state, the returned task will also complete in a <xref:System.Threading.Tasks.TaskStatus?displayProperty=fullName> state,  where its exceptions will contain the aggregation of the set of unwrapped exceptions from each of the supplied tasks.  
+ If any of the supplied tasks completes in a faulted state, the returned task will also complete in a <xref:System.Threading.Tasks.TaskStatus.Faulted> state,  where its exceptions will contain the aggregation of the set of unwrapped exceptions from each of the supplied tasks.  
```

Comment:
The xref should be `<xref:System.Threading.Tasks.TaskStatus.Faulted?displayProperty=fullName>`.